command/operator_keyring.go

```diff
@@ -88,6 +88,9 @@ func (c *OperatorKeyringCommand) Run(args []string) int {
 		return 1
 	}
 
+	c.Ui.Error(`nomad operator keyring is deprecated and will be removed in Nomad 1.5.0.
+Use nomad operator gossip keyring`)
+
```

Comment:
To keep the deprecation notice consistent with the ordinary deprecated commands?
```suggestion
	c.Ui.Warn(wrapAtLength("WARNING! The \"nomad operator keyring\" command " +
		"is deprecated. Please use \"nomad operator gossip keyring\" instead. " +
		"This command will be removed in Nomad 1.5.0."))
	c.Ui.Warn("")
```